mountpoint-s3/src/fs.rs

```diff
@@ -300,6 +301,7 @@ where
             inode: lookup.inode,
             full_key,
             object_size: lookup.stat.size as u64,
+            etag: lookup.stat.etag,
```

Comment:
If you do my earlier comment about making this field not be optional, then we should fail the `open` request if the object doesn't have an ETag for some reason.